www/i18n/en.json

```diff
@@ -170,10 +170,10 @@
         "purpose": "Purpose",
         "survey": "Details",
         "untracked-time-range": "Untracked: {{start}} - {{end}}",
-        "unlabeled": "All Unlabeled",
+        "unlabeled": "All Unlabeled",
         "invalid-ebike": "Invalid",
-        "to-label": "To Label",
-        "show-all": "All Trips",
+        "to-label": "To Label",
+        "show-all": "All Trips",
```

Comment:
These were non-breaking spaces (NBSP) and I replaced them with normal spaces.

These filter names are now translated in the JS instead of the HTML. The NBSP were parsing incorrectly and showing something like `To&nbsp;Label`.

The other languages did not have non-breaking spaces.

Reply:
Yeah I noticed that you had a comment about &nbsp; but didn't realize that they were rendered as spaces in the diffs.